tests/test_reset.py

```diff
@@ -0,0 +1,154 @@
+"""
+This file was adapted from a setup file given in 
+the https://github.com/microsoft/AirSim-Drone-Racing-Lab repositiry
+It simply loads a level and starts a race against a baseline drone
+to validate that everything has been set up properly.
+"""
+
+import airsimdroneracinglab
+import threading
+import time
+
+
+class ReproduceResetRaceCondition:
+    """
+    Contains vaious functions for interacting with the ADRL unreal environment
+    """
+
+    def __init__(self, drone_name="drone_1"):
+        """
+        Initializes class variables
+
+        Args:
+        -----
+            drone_name (string) = Name of user controlled drone (defaults to 'drone_1')
```

Comment:
Indicate type like in normal code
```suggestion
            drone_name: string = Name of user controlled drone (defaults to 'drone_1')
```